WhatIsASprint.md

```diff
@@ -0,0 +1,23 @@
+
+# **What is a Sprint?**
+
+
+A **_sprint_** is a set period of time broken out from a project's overall timeline with associated tasks to be completed during that period of time. That list of tasks is referred to as the sprint's <b>Product Backlog</b>.
+
+
+A *sprint* allows a larger timeline to be easily broken into smaller, more digestible pieces in order to focus effort from all stakeholders on a few logically scheduled tasks at a time.
```

Comment:
This is not describing sprint as the process but as a tool or goal. The ideology behind a sprint allows. I have no issue adding this clarification if you still feel it is needed. Everything discussed it a product of human decision making, therefore that is not a valid comment to add.